tests/cases/fourslash/autoImportTypeImport4.ts

```diff
@@ -23,27 +23,28 @@
 
 // @Filename: /index0.ts
 //// import { A, B, C } from "./exports1";
-//// a/*0*/;
+//// a/*0*//*0a*/;
 //// b;
 
 // @Filename: /index1.ts
 //// import { A, B, C, type Y, type Z } from "./exports1";
-//// a/*1*/;
+//// a/*1*//*1a*//*1b*//*1c*/;
 //// b;
 
 // @Filename: /index2.ts
 //// import { A, a, B, b, type Y, type Z } from "./exports1";
 //// import { E } from "./exports2";
-//// d/*2*/
+//// d/*2*//*2a*//*2b*//*2c*/
 
 // addition of correctly sorted type imports should not affect behavior as shown in autoImportSortCaseSensitivity1.ts
 goTo.marker("0");
 verify.importFixAtPosition([
-    `import { A, B, C, a } from "./exports1";\na;\nb;`,
-    `import { A, B, C, b } from "./exports1";\na;\nb;`,
+    `import { a, A, B, C } from "./exports1";\na;\nb;`,
+    `import { A, b, B, C } from "./exports1";\na;\nb;`,
 ],
     /*errorCode*/ undefined,
     { organizeImportsTypeOrder : "last" });
+goTo.marker("0a");

```

Comment:
What is with the extra navigation here? Aren't these all at the same positions?

Reply:
Sorry, just saw this comment. It helps to see which of the tests fail, as opposed to just reading "test failed at marker 0" for all of the tests. I can change it back if it's going to make the test much slower?

Reply:
No, that totally makes sense, I just haven't seen any examples of this.

Not for this PR, but this makes me wonder if we should have the fourslash harness keep track of how many times it runs each function to better indicate what it's doing, or include the line number, etc. I think some of that already happens in stack traces?